lib/registry.js

```diff
@@ -43,14 +43,14 @@ class Registry {
 				}
 			}
 
-			let labels = '';
+			const labels = [];
 			for (const key of Object.keys(val.labels)) {
-				labels += `${key}="${escapeLabelValue(val.labels[key])}",`;
+				labels.push(`${key}="${escapeLabelValue(val.labels[key])}"`);
 			}
 
 			let metricName = val.metricName || item.name;
 			if (labels) {
-				metricName += `{${labels.substring(0, labels.length - 1)}}`;
+				metricName += `{${labels.join(',')}}`;
```

Comment:
Thanks for the PRs!

Unfortunately this approach is slower:

```
✗ registry ➭ metrics#1 with 64 is 11.13% slower.
⚠ registry ➭ metrics#2 with 8 is 7.170% acceptably slower.
⚠ registry ➭ metrics#2 with 4 and 2 with 2 is 0.4725% acceptably slower.
⚠ registry ➭ metrics#2 with 2 and 2 with 4 is 8.324% acceptably slower.
⚠ registry ➭ metrics#6 with 2 is 6.719% acceptably slower.
```

If you want to change it, could you try out the approach outlined in https://github.com/siimon/prom-client/pull/235#discussion_r244376041? You can run `yarn/npm benchmarks` while on your branch to compare against the current release.